todo_app/data/trello_items.py

```diff
@@ -0,0 +1,121 @@
+import requests
+import json
+import os
+
+class Cards:
```

Comment:
Usually we'd name this in the singular form if we're referring to a single object, if I saw I was creating "Cards" I'd assume it represents multiple whereas we want to represent one "Card"